src/main/java/com/backbase/ct/bbfuel/client/user/UserIntegrationRestClient.java

```diff
@@ -29,10 +36,20 @@ public void init() {
     }
 
     public void ingestUserAndLogResponse(UsersPostRequestBody user) {
-        Response response = ingestUser(user);
 
-        if (isBadRequestException(response, "User already exists")) {
+        Response response;
+
+        if (this.globalProperties.getBoolean(PROPERTY_IDENTITY_FEATURE_TOGGLE)) {
+            response = importUserIdentity(user);
+        } else {
+            response = ingestUser(user);
+        }
+
+        if (isBadRequestException(response, "User already exists") || isConflictException(response,"User already exists")) {
```

Comment:
```suggestion
        if (isBadRequestException(response, "User already exists") || isConflictException(response, "User already exists")) {
```

Just a formatting issue